sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -711,9 +711,9 @@
                         "maxLength": "",
                         "minWords": "",
                         "maxWords": "",
-                        "pattern": "",
-                        "customMessage": "",
-                        "custom": "valid = /^(11|09|52|01|03|05|16|23|24|25|42|27|30|38|40|45|50|54|55|13|28|31|44|60|04|10|14|15|29|41|26|22|43|51|39|12|21|46|47|48){1}\\.[a-zA-Z]{4}$/.test(input) ? true : 'Incorrect Format';",
+                        "pattern": "^(11|09|52|01|03|05|16|23|24|25|42|27|30|38|40|45|50|54|55|13|28|31|44|60|04|10|14|15|29|41|26|22|43|51|39|12|21|46|47|48){1}\\.[0-9]{4}",
```

Comment:
I believe that there is a "$" missing at the end of the regex. Please change to below. In the way that it is, it will accept inputs like 10.123456789.
```
^(11|09|52|01|03|05|16|23|24|25|42|27|30|38|40|45|50|54|55|13|28|31|44|60|04|10|14|15|29|41|26|22|43|51|39|12|21|46|47|48){1}\\.[0-9]{4}$
```

Reply:
Added it.